Makefile

```diff
@@ -93,6 +93,15 @@ release-publish-local:
 	make release-local
 	make publish-local
 
+clone-swiftline:
+	@echo "Removing old Swiftline"
+	@echo ""
+	rm -rf SwiftLineTemp
+
+	@echo "Clone new Swiftline from oarrabi/linux! branch"
+	git clone -b oarrabi/linux! https://github.com/oarrabi/Swiftline.git SwiftLineTemp
```

Comment:
Are we commiting to maintaing this branch green. I think we shoudl use another name. Like 'vendored'.